libcxx/test/std/numerics/numeric.ops/numeric.ops.sat/add_sat.pass.cpp

```diff
@@ -25,8 +25,7 @@ constexpr bool test_signed() {
   constexpr auto minVal = std::numeric_limits<IntegerT>::min();
   constexpr auto maxVal = std::numeric_limits<IntegerT>::max();
 
-  // TODO(LLVM-20) remove [[maybe_unused]]  since all supported compilers support "Placeholder variables with no name"
-  [[maybe_unused]] std::same_as<IntegerT> decltype(auto) _ = std::add_sat(minVal, maxVal);
+  std::same_as<IntegerT> decltype(auto) _ = std::add_sat(minVal, maxVal);
```

Comment:
Please don't do this here I have a complete an proper PR here: https://github.com/llvm/llvm-project/pull/101826

Reply:
Thanks for the heads up! Seems like you are already one step ahead of me. Let's limit this PR only to the update of the CI config and bring your changes in. Afterwards, I will create a PR including adjustments for libc++ tests and implementations that you have not covered.